tests/parsers/locate.py

```diff
@@ -34,12 +34,31 @@ def testParseFile(self):
 
     expected_event_values = {
         'data_type': 'linux:locate_database:entry',
-        'paths': ['/home/user/temp'],
+        'path': '/home/user/temp',
+        'contents': ['1', '2', '3', 'À'],
         'written_time': '2021-07-09T04:36:19.606373200+00:00'}
 
     event_data = storage_writer.GetAttributeContainerByIndex('event_data', 0)
     self.CheckEventData(event_data, expected_event_values)
 
+    expected_event_values = {
+        'data_type': 'linux:locate_database:entry',
+        'path': '/home/user/temp/1',
+        'contents': ['1a.txt', '1b.txt'],
+        'written_time': '2021-07-09T04:11:07.438810500+00:00'}
+
+    event_data = storage_writer.GetAttributeContainerByIndex('event_data', 1)
+    self.CheckEventData(event_data, expected_event_values)
+
```

Comment:
Maybe add a comment that you're testing a path without entries

Reply:
Done